from elvia_louvre.errors import LouvreException


class LouvreLoggerSetupError(LouvreException):
    pass
